recipes/mongo-cxx-driver/all/conanfile.py

```diff
@@ -0,0 +1,188 @@
+
+        if self.settings.os == "Windows":
+            for dll_file in glob.glob(os.path.join(self.package_folder, "bin", "*.dll")):
+                if os.path.basename(dll_file).startswith(("concrt", "msvcp", "vcruntime")):
+                    os.remove(dll_file)
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+    def package_info(self):
+        # FIXME: two CMake module/config files should be generated (mongocxx-config.cmake and bsoncxx-config.cmake),
+        # but it can't be modeled right now.
+        self.cpp_info.filenames["cmake_find_package"] = "mongocxx"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "mongocxx"
+        self.cpp_info.names["cmake_find_package"] = "mongo"
+        self.cpp_info.names["cmake_find_package_multi"] = "mongo"
+
+        # mongocxx
+        self.cpp_info.components["mongocxx"].names["cmake_find_package"] = "mongocxx_shared" if self.options.shared else "mongocxx_static"
+        self.cpp_info.components["mongocxx"].names["cmake_find_package_multi"] = "mongocxx_shared" if self.options.shared else "mongocxx_static"
+        self.cpp_info.components["mongocxx"].names["pkg_config"] = "libmongocxx"
+        self.cpp_info.components["mongocxx"].includedirs = [os.path.join("include", "lib", "v_noabi", "mongocxx")]
+        self.cpp_info.components["mongocxx"].libs = ["mongocxx" if self.options.shared else "mongocxx-static"]
+        if not self.options.shared:
+            self.cpp_info.components["mongocxx"].defines.append("MONGOCXX_STATIC")
+        if self.options.with_ssl:
+            self.cpp_info.components["mongocxx"].defines.append("MONGOCXX_ENABLE_SSL")
+        self.cpp_info.components["mongocxx"].requires = ["mongo-c-driver::mongoc", "bsoncxx"]
+
+        # bsoncxx
+        self.cpp_info.components["bsoncxx"].names["cmake_find_package"] = "bsoncxx_shared" if self.options.shared else "bsoncxx_static"
+        self.cpp_info.components["bsoncxx"].names["cmake_find_package_multi"] = "bsoncxx_shared" if self.options.shared else "bsoncxx_static"
+        self.cpp_info.components["bsoncxx"].names["pkg_config"] = "libbsoncxx" if self.options.shared else "libbsoncxx-static"
+        self.cpp_info.components["bsoncxx"].includedirs = [os.path.join("include", "lib", "v_noabi", "bsoncxx")]
+        self.cpp_info.components["bsoncxx"].libs = ["bsoncxx" if self.options.shared else "bsoncxx-static"]
+        if not self.options.shared:
+            self.cpp_info.components["bsoncxx"].defines = ["BSONCXX_STATIC"]
+        self.cpp_info.components["bsoncxx"].requires = ["mongo-c-driver::bson"]
+        if self.options.polyfill == "boost":
+            self.cpp_info.components["bsoncxx"].requires.append("boost::boost")
+        if self.options.polyfill == "mnmlstc":
+            self.cpp_info.components["bsoncxx"].includedirs.append(os.path.join("include", "bsoncxx", "third_party", "mnmlstc"))
```

Comment:
```suggestion
```

We have decided to raise if `polyfill == "mnmlstc"` because this dependency is not yet in CCI, so headers of vendored mnmlstc can't be installed I guess.

Reply:
Ok, will delete it.